packages/react-jss/src/memoize-one.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+function memoize<Arg, Return>(fn: (arg: Arg) => Return) {
```

Comment:
btw why do you like named functions so much? by the latest spec a function expression assigned to a variable has a function name as well, derived from the variable name. So there is no problem with debugging.

I don't like named functions because of the implicit hoisting, its a generally leaky concept.

Reply:
its like var and even worse, no one is using var any more and named functions are even worse, because they can be called before they are defined and once people start relying on this, code gets messy.

Reply:
Personally, it makes it easier to differentiate between variables and I'm always using eslint or tslint where I disallow to not use something before it's not defined.